SAPex/Services/InitService.cs

```diff
@@ -0,0 +1,143 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using DbMigrations.Data;
+using DbMigrations.EntityModels;
+using Microsoft.EntityFrameworkCore;
+
+namespace SAPex.Services
+{
+    public class InitService
```

Comment:
Please create the "Helpers" folder in the BLL project and move the "Initial" service into this folder. Please rename class like "TestDataHelper" or "InitialDataHelper".

Reply:
done